src/content/docs/extensions/algos/index.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Graph algorithms
+---
+
+The graph algorithms extension package allows you to directly run popular graph algorithms
+like PageRank, connected components and Louvain on your graph data stored in Kuzu. Using this extension,
+you do not need to export your data to specialized graph analytics tools like NetworkX (at
+least for the algorithms that are supported by the extension). The algorithms run natively
+in Kuzu, which also allows you to scale to very large graphs!
+
+Graph algorithms are useful tools for extracting meaningful insights from connected data.
+Whether you're detecting fraud patterns in financial transactions, optimizing supply chain networks,
+or analyzing social media interactions, these algorithms help you understand complex relationships and
+make data-driven decisions. The following sections describe how to use the graph algorithms extension
+in Kuzu.
+
+## Project graph
```

Comment:
This subsection needs to be written in a more generic way as it is not specific to algorithms only, our filtered search on vector index also relies on this.

Reply:
Will other functionalities than vector search use this? Maybe we can wait till the next release to reorg?

Reply:
Not yet. We can wait a bit on reorg here.